kairon/shared/actions/utils.py

```diff
@@ -904,6 +904,53 @@ def prepare_hubspot_form_request(tracker, fields: list, bot: Text):
     def get_basic_auth_str(username: Text, password: Text):
         return requests.auth._basic_auth_str(username, password)
 
+    @staticmethod
+    def prepare_flow_body(flow_action_config: dict, tracker: Tracker):
+        http_url = Utility.environment["flow"]["url"]
+        header_key = Utility.environment["flow"]["headers"]["key"]
+        header_value = Utility.environment["flow"]["headers"]["value"]
+        headers = {header_key: header_value}
+        flow_body = {
+            "recipient_type": "individual",
+            "messaging_product": "whatsapp",
+            "type": "interactive",
+            "interactive": {
+                "type": "flow",
+                "action": {
+                    "name": "flow",
+                    "parameters": {
+                        "mode": "published",
+                        "flow_message_version": "3",
+                        "flow_token": "AQAAAAACS5FpgQ_cAAAAAD0QI3s.",
+                        "flow_action": "navigate",
```

Comment:
default value for flow_action will be 'navigate' but dont add this as hard coded, flow action can be data_exchange